packages/core/src/index.ts

```diff
@@ -394,6 +394,13 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
     return this._libp2p.multiaddrs
   }
 
+  /*
+   * List the addresses on which the node is listening
+   */
+  public getListeningAddresses(): Multiaddr[] {
+    return this._libp2p.addressManager.getListenAddrs()
```

Comment:
What is the difference between listening addresses and addresses?

Reply:
see #tech chat